lib/services/ngsi/ngsiService.js

```diff
@@ -575,10 +687,21 @@ function formatAsNGSILD(json){
             case 'type':
                 obj[key] = json[key];
                 break;
+           case constants.TIMESTAMP_ATTRIBUTE:
+                /*               
```

Comment:
Leftover? Code to ve activated in a next PR?

Reply:
Fixed. Replaced with comment - 3d72455